css/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,17 @@
+.moon-badge-overlay-support .moon-image-client {
+	color: white;
+	padding: 0;
+	background-color: rgba(0, 0, 0, 0.5);
+	.moon-icon {
+		width: 36px;
+		height: 36px;
```

Comment:
If you're going to change the size of an icon, you'll also need to change the size of the line-height, so the font-icons stay vertically centered in their frame:
```
line-height: 36px;
```